lib/philomena_web/views/setting_view.ex

```diff
@@ -1,18 +1,50 @@
 defmodule PhilomenaWeb.SettingView do
   use PhilomenaWeb, :view
 
-  def theme_options do
+  def themes do
     [
-      [
-        key: "Default",
-        value: "default",
-        data: [theme_path: ~p"/css/default.css"]
-      ],
-      [key: "Dark", value: "dark", data: [theme_path: ~p"/css/dark.css"]],
-      [key: "Red", value: "red", data: [theme_path: ~p"/css/red.css"]]
+      Dark: "dark",
+      Light: "light"
     ]
   end
 
+  def theme_colors do
+    [
+      Red: "red",
+      Orange: "orange",
+      Yellow: "yellow",
+      Green: "green",
+      Blue: "blue",
+      Purple: "purple",
+      Cyan: "cyan",
+      Pink: "pink",
+      "Silver/Charcoal": "silver"
```

Comment:
I would prefer to stick to one name, Silver

Reply:
I don't think that's appropriate considering they look entirely different depending on whether you're using the dark or light theme.

Reply:
How about "Gray" as a shared name then?

Reply:
sure